pkg/volume/csi/csi_client.go

```diff
@@ -72,9 +72,17 @@ type csiClient interface {
 		secrets map[string]string,
 		volumeContext map[string]string,
 	) error
+
+	NodeGetVolumeStats(
+		ctx context.Context,
+		volID string,
+		targetPath string,
+	) (map[string]usageCount,
+		error)
```

Comment:
I do not think we need to split the return values on 2 lines.

Reply:
Done.